fabfile.py

```diff
@@ -89,7 +92,7 @@ def setup():
     print "***** Please choose a passphrase and type it at the three prompts."
```

Comment:
I haven't tried it myself, but based on your commit message this should say "two prompts" now?